qiskit_experiments/quantum_volume/qv_experiment.py

```diff
@@ -0,0 +1,249 @@
+
+        # Add Jobs to ExperimentData
+        experiment_data.add_data(job)
+
+        sim_data = self._get_ideal_data(ideal_circuits, **run_opts)
+        # Add Jobs to the simulation data
+        simulation_data.add_data(sim_data)
+
+        # Queue analysis of data for when job is finished
+        if analysis and self.__analysis_class__ is not None:
+            self.run_analysis(experiment_data, simulation_data=simulation_data)
+
+        self._simulation_data = simulation_data
+        # Return the ExperimentData future
+        return experiment_data
+
+    def add_trials(self, additional_trials):
+        """
+        Add more trials to the experiment
+        Args:
+            additional_trials (int): The amount of trials to add
+        """
+        new_trials = self.experiment_options.trials + additional_trials
+        self.set_experiment_options(trials=new_trials)
+
+    @property
+    def trials(self):
+        """Return number of trials in the experiment"""
+        return self.experiment_options.trials
+
+    @property
+    def simulation_data(self):
+        """Return the ideal data of the experiment"""
+        return self._simulation_data
+
+    def _get_ideal_data(self, circuits, **run_options):
+        """
+        In case the user does not have Aer installed - use Terra to calculate the ideal state
+        Args:
+            circuits: the circuits to extract the ideal data from
+        Returns:
+            dict: data object with the circuit's metadata
+                  and the probability for each state in each circuit
+        """
+        if self._simulation_backend:
+            for circuit in circuits:
+                circuit.save_probabilities()
+            return execute(circuits, backend=self._simulation_backend, **run_options)
+        else:
+            from qiskit.quantum_info import Statevector
+
+            sim_obj = []
+            for circuit in circuits:
+                state_vector = Statevector(circuit)
+                sim_data = {
+                    "probabilities": state_vector.probabilities(),
+                    "metadata": circuit.metadata,
+                }
+                sim_obj.append(sim_data)
+            return sim_obj
+
+    def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
+        """Return a list of QV circuits, without the measurement instruction
+        Args:
+            backend (Backend): Optional, a backend object.
+        Returns:
+            List[QuantumCircuit]: A list of :class:`QuantumCircuit`s.
+        """
+        circuits = []
+        depth = self._num_qubits
+
+        # Continue the trials numbers from previous experiments runs
+        for trial in range(self._previous_trials + 1, self.experiment_options.trials + 1):
+            qv_circ = QuantumVolume(depth, depth, seed=self._rng)
+            qv_circ.measure_active()
+            qv_circ.metadata = {
+                "experiment_type": self._type,
```

Comment:
I decided against using `_postprocess_transpiled_circuits` since the probabilities should be computed from the pre-transpiled circuits, and otherwise we'd have to keep the pre-transpiled circuits until this method is called.